delegation-toolkit/concepts/smart-accounts.md

```diff
@@ -37,21 +37,21 @@ ERC-4337 introduces the following concepts:
 
 ## Smart account implementation types
 
-The MetaMask Delegation Toolkit supports three types of MetaMask Smart Accounts, each offering unique features and use cases.
+The MetaMask Delegation Toolkit supports three types of MetaMask Smart Accounts, each offering different signature schemes for verifying access to the smart account.
 
 See [Create a smart account](../guides/smart-accounts/create-smart-account.md) to learn how to use these different account types.
 
 ### Hybrid smart account
 
-The Hybrid smart account is a flexible implementation that supports both an externally owned account (EOA) owner and any number of P256 (passkey) signers.
+The Hybrid smart account is a flexible implementation that supports both an externally owned account (EOA) owner and any number of passkey (WebAuthn) signers.
 You can configure any of these signers as the signatory, and use them to sign any data, including user operations, on behalf of the smart account.
 
 This type is referenced in the toolkit as `Implementation.Hybrid`.
 
 ### Multisig smart account
 
 The Multisig smart account is an implementation that supports multiple signers with a configurable threshold for valid signatures, allowing for enhanced security and flexibility in account management.
-The signatory must have at least as many signers include as the threshold is configured for the account.
+The signatory, which signs on behalf of the smart account, must have at least as many signers as the configured threshold.
```

Comment:
This is not correct, signatory doesn't need to have n(threshold) number of signers configured. In ideal world, we won't even recommend having signatory with n(threshold) numbers of signers configured, as it defeats the purpose. 

Maybe we can rephrase, "A valid signature requires signatures from at least the number of signers specified by the threshold."